clients/drcachesim/tracer/tracer.cpp

```diff
@@ -2336,7 +2336,14 @@ drmemtrace_client_main(client_id_t id, int argc, const char *argv[])
             DR_ASSERT(false);
```

Comment:
> i#6548 drmemtrace does not create the required pipe file if not present.

nit: We put a colon after the issue number in the title: `s/i#6548 d/i#6548: d/`